src/ansi-c/expr2c.cpp

```diff
@@ -862,6 +842,58 @@ std::string expr2ct::convert_array_type(
 
 /*******************************************************************\
 
+Function: expr2ct::convert_pointer_type
+
+  Inputs:
+          src - The pointer type to convert
+          qualifiers - A list of the qualifiers of the pointer declaration
+          declarator_str - The declaration string we will build
+          toggle - Activate the pointer -> array treating behavior
+
+ Outputs: A C-like type declaration of a pointer
+
+ Purpose: To generate a C-like type declaration of an pointer
+\*******************************************************************/
+
+std::string expr2ct::convert_pointer_type(
+  const typet &src,
+  const c_qualifierst &qualifiers,
+  const std::string &declarator,
+  bool toggle)
+{
+  c_qualifierst new_qualifiers(qualifiers);
```

Comment:
I am still confused about this - I thought we worked out it wasn't needed. I'm not really sure what the difference between `qualifiers` and `new_qualifiers`. The old `q` seems to be generated from the `qualifiers` that is passed in.